src/HealthChecks.SendGrid/HealthChecks.SendGrid.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetStandardTargetVersion)</TargetFramework>
+    <PackageTags>HealthCheck;SendGrid</PackageTags>
+    <Description>HealthChecks.SendGrid is the health check package for SendGrid.</Description>
+    <Version>$(HealthCheckSendGrid)</Version>
+    <LangVersion>latest</LangVersion>
+    <PackageId>AspNetCore.HealthChecks.SendGrid</PackageId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="$(MicrosoftExtensionsDiagnosticsHealthChecks)" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkGithub)">
```

Comment:
on latest code  ( your pr )  we remove this dependy on healthchecks project because is alread added on Directory.Build.props